### PR TITLE
return property title in header, association data output

### PR DESCRIPTION
**Pull request by jokedurnez**
With this, the column headers are much clearer, and we have a way to change the column names.

Also includes adding association data into the exported data.

image

### CODE REVIEW

File: src/main/java/com/openlattice/chronicle/services/ChronicleServiceImpl.java [hunk 1]
```diff
@@ -983,16 +984,26 @@ public void refreshStudyInformation() {
...
+            // create a dictionary from property fqn to title of
+            EntityType appDataET = edmApi.getEntityType( entitySetsApi.getEntitySet( srcEntitySetId ).getEntityTypeId() );
+            Map<String, String> columnNameDictionary = appDataET.getProperties()
+                    .stream()
+                    .collect(
+                            Collectors.toMap(
+                                    k -> edmApi.getPropertyType( k ).getType().getFullQualifiedNameAsString(),
+                                    k -> entitySetsApi.getEntitySetPropertyMetadata( srcEntitySetId, k ).getTitle()
+                            )
+                    );
```
Comment (UnsungHero97) [inline src/main/java/com/openlattice/chronicle/services/ChronicleServiceImpl.java:1005]: There's 12 properties for this entity type, which means this is making 24 API requests to gather all this data. This doesn't seem ideal.

Comment (jokedurnez) [reply src/main/java/com/openlattice/chronicle/services/ChronicleServiceImpl.java]: Done, only 2 requests now.

File: src/main/java/com/openlattice/chronicle/constants/OutputConstants.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package com.openlattice.chronicle.constants;
+
+public class OutputConstants {
+
+    private OutputConstants() {
+    }
+
+    // prefix for column names
+    public static final String ENTITY_PREFIX = "app_";
```
Comment (UnsungHero97) [inline src/main/java/com/openlattice/chronicle/constants/OutputConstants.java:9]: Rename to `APP_PREFIX`.

File: src/main/java/com/openlattice/chronicle/constants/OutputConstants.java [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+    public static final String ASSOCIATION_PREFIX = "user_";
```
Comment (UnsungHero97) [inline src/main/java/com/openlattice/chronicle/constants/OutputConstants.java:10]: Rename to `USER_PREFIX`.

**Pull request merged**